src/sbt-test/sbt-autoversion/conventional-commits/version.sbt

```diff
@@ -0,0 +1 @@
+ThisBuild / version := "0.0.0"
```

Comment:
You've added sbt-dynver .. doesn't that plugin deprecate the setting of `version` directly?

Reply:
I've only added dynver to _this_ plugin's build (not dogfooding quite yet 😱 🙉)  - it shouldn't affect clients of this plugin, which for the most part still use `version.sbt` by way of sbt-release (as in this test project).